README.md

```diff
@@ -417,42 +235,7 @@ provide a more detailed explanation:
     *   Create migration: `rails g migration [remove_parent_id_from_](table)`
     *   Add to migration: `remove_column [table], :parent_id`
     *   Migrate your database: `rake db:migrate`
-
-# Integrity checking and restoration
-
-I don't see any way Ancestry tree integrity could get compromised without
-explicitly setting cyclic parents or invalid ancestry and circumventing
-validation with update_attribute. If you do, please let me know.
-
-Ancestry includes some methods for detecting integrity problems and restoring
-integrity just to be sure. To check integrity, use:
-`[Model].check_ancestry_integrity!`. An AncestryIntegrityException will be
-raised if there are any problems. You can also specify :report => :list to
-return an array of exceptions or :report => :echo to echo any error messages.
-To restore integrity use: `[Model].restore_ancestry_integrity!`.
-
-For example, from IRB:
-
-```
->> stinky = TreeNode.create :name => 'Stinky'
-$  #<TreeNode id: 1, name: "Stinky", ancestry: nil>
->> squeeky = TreeNode.create :name => 'Squeeky', :parent => stinky
-$  #<TreeNode id: 2, name: "Squeeky", ancestry: "1">
->> stinky.update_attribute :parent, squeeky
-$  true
->> TreeNode.all
-$  [#<TreeNode id: 1, name: "Stinky", ancestry: "1/2">, #<TreeNode id: 2, name: "Squeeky", ancestry: "1/2/1">]
->> TreeNode.check_ancestry_integrity!
-!! Ancestry::AncestryIntegrityException: Conflicting parent id in node 1: 2 for node 1, expecting nil
->> TreeNode.restore_ancestry_integrity!
-$  [#<TreeNode id: 1, name: "Stinky", ancestry: 2>, #<TreeNode id: 2, name: "Squeeky", ancestry: nil>]
-```
-
-Additionally, if you think something is wrong with your depth cache:
-
-```
->> TreeNode.rebuild_depth_cache!
-```
+</details>
 
 # Running Tests
```

Comment:
if we want we can have a footnote around appraisals

but this is an ancestry development thing. not sure if that is good or mucking it up

thinking we should have appraisal back in the gemspec as a development dependency?


either way, these docs should only have appraisal install ; appraisal rake test
suggest `wwtd --local` (also not a declared dependency)